src/index.js

```diff
@@ -0,0 +1,42 @@
+import './style.css';
+import navBar from './_navbar';
+import home from './home';
+import menuSection from './food';
+import contactSection from './contact';
+
+const body = document.querySelector('body');
+const mainDiv = document.querySelector('#content');
+body.prepend(navBar());
+
+mainDiv.appendChild(home());
+
+// Functions for menus
+const goToIndex = () => {
+  mainDiv.textContent = '';
+  mainDiv.appendChild(home());
+};
+
+// const goToAbout = () => {
+//   mainDiv.textContent = '';
+//   mainDiv.appendChild(loadAboutUs());
+// };
```

Comment:
- [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))